cmd/podman/artifact/pull.go

```diff
@@ -0,0 +1,194 @@
+package artifact
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/containers/buildah/pkg/cli"
+	"github.com/containers/common/pkg/auth"
+	"github.com/containers/common/pkg/completion"
+	"github.com/containers/image/v5/types"
+	"github.com/containers/podman/v5/cmd/podman/common"
+	"github.com/containers/podman/v5/cmd/podman/registry"
+	"github.com/containers/podman/v5/pkg/domain/entities"
+	"github.com/containers/podman/v5/pkg/util"
+	"github.com/spf13/cobra"
+)
+
+// pullOptionsWrapper wraps entities.ImagePullOptions and prevents leaking
+// CLI-only fields into the API types.
+type pullOptionsWrapper struct {
+	entities.ArtifactPullOptions
+	TLSVerifyCLI   bool // CLI only
+	CredentialsCLI string
+	DecryptionKeys []string
+}
+
+var (
+	pullOptions     = pullOptionsWrapper{}
+	pullDescription = `Pulls an artifact from a registry and stores it locally.`
+
+	pullCmd = &cobra.Command{
+		Use:   "pull [options] ARTIFACT",
+		Short: "Pull an OCI artifact",
+		Long:  pullDescription,
+		RunE:  artifactPull,
+		//PersistentPreRunE: devOnly,
```

Comment:
devOnly is not defined anywhere else, either remove or comment a TODO on what this means

Reply:
yeah leftover turd where i was trying to omit a warning for expiremental, thanks!